coralproject/talk
# [CORL-2294] ReplyCommentForm permanently autofocuses

## What does this PR do?
- Fixes bug where ReplyCommentForm would autofocus permanently

## how do test
- Test with `COMMENT_SEEN` enabled.
- Open a reply form
- Now click on a different comment to put focus there
- Notice that ReplyCommentForm does not retake focus anymore
